Cargo.toml

```diff
@@ -13,16 +13,13 @@ exclude = ["media"]
 
 [features]
 # FIXME: remove "cache-me" that was added by mistake here
```

Comment:
can you fix this too, while we are at it?

Reply:
The fix got lost it seems

Reply:
Though given that `dispatching2` feature got removed and the new dispatching requires `cache-me`, maybe it needs to be always enabled?...

```diff,toml
-teloxide-core = { version = "0.4", default-features = false }
+teloxide-core = { version = "0.4", default-features = false, features = ["teloxide-core/cache-me"] }
...
throttle = ["teloxide-core/throttle"]
-cache-me = ["teloxide-core/cache_me"]
trace-adaptor = ["teloxide-core/trace_adaptor"]
```

Reply:
Actually, why it's even needed? Dispatcher can simply store `Me`, instead of `CacheMe<R>` it would be easier & """"more performant""".

(you don't need to fix it here, I'll make a follow-up PR)

Reply:
Also we should add CI job that tests with `--no-default-features`

Reply:
Okay, sounds reasonable, let's make it in a separate PR.